### PR TITLE
Include project name in test case name

### PR DESCRIPTION
**Pull request by gatukgl**
**Changes proposed in this pull request:**
- Include project name as a prefix for test case name
- Setup prettier for cypress test

**Example result:**
- project: `Simplesat`
- test case name: `test case 1`
image

### CODE REVIEW

File: tests/cypress/integration/testruns.spec.js [hunk 1]
```diff
@@ -15,117 +15,89 @@ context('Testruns', () => {
...
-      .select('passed')
-    cy.get('input.default')
-      .click()
-    cy.contains(project_name)
-      .click()
+    cy.get('div.app-testruns a.addlink').click()
+    cy.get('#id_name').type(`Test new test run ${random_string}`)
+    cy.get('#id_project').select(project_name)
+    cy.get('#id_testinstance_set-0-testcase').select(
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 2]
```diff
@@ -15,117 +15,89 @@ context('Testruns', () => {
...
-      .type(random_string)
-    cy.get('#changelist-search > div > [type="submit"]')
-      .click()
-    cy.get(':nth-child(1) > .field-name')
-      .should('have.text', `Test new test run ${random_string}`)
+    cy.get('div.app-testruns a.addlink').click()
+    cy.get('#id_name').type(`Test new test run ${random_string}`)
+    cy.get('#id_project').select(project_name)
+    cy.get('#id_testinstance_set-0-testcase').select(
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 3]
```diff
@@ -15,117 +15,89 @@ context('Testruns', () => {
...
-      .select('passed')
-    cy.get('input.default')
-      .click()
-    cy.get(':nth-child(1) > .field-id > a')
-      .click()
+    cy.get('div.app-testruns a.addlink').click()
+    cy.get('#id_name').type(`Test new test run ${random_string}`)
+    cy.get('#id_project').select(project_name)
+    cy.get('#id_testinstance_set-0-testcase').select(
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 4]
```diff
@@ -15,117 +15,89 @@ context('Testruns', () => {
...
-      .click()
-    cy.get(':nth-child(1) > .field-name')
-      .should('have.text', `Test new test run ${random_string}`)
-    cy.get(':nth-child(1) > .field-project')
-      .should('have.text', project_name)
+    cy.get('div.app-testruns a.addlink').click()
+    cy.get('#id_name').type(`Test new test run ${random_string}`)
+    cy.get('#id_project').select(project_name)
+    cy.get('#id_testinstance_set-0-testcase').select(
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

**Comment by gatukgl** [comment]
will fix conflict  ja

**Comment by yothinix** [comment]
@gatukgl Conflict 😛

File: tests/cypress/integration/testinstances.spec.js [hunk 5]
```diff
@@ -17,101 +17,104 @@ context('Test Instances', () => {
...
-    cy.get('.btn-info')
-      .click()
-    cy.get(':nth-child(1) > .field-testcase')
-      .should('have.text', testcase_name)
-    cy.get(':nth-child(1) > .field-status')
-      .should('have.text', 'Passed')
+    cy.get('.btn-info').click()
+    cy.get(':nth-child(1) > .field-testcase').should(
+      'have.text',
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testinstances.spec.js]: This is what I really changed.

File: tests/cypress/integration/testinstances.spec.js [hunk 6]
```diff
@@ -17,101 +17,104 @@ context('Test Instances', () => {
...
-    cy.get('#changelist-search > div > [type="submit"]')
-      .click()
-    cy.get(':nth-child(1) > .field-testcase')
-      .should('have.text', testcase_name)
-    cy.get(':nth-child(1) > .field-status')
-      .should('have.text', 'Passed')
+    cy.get('#changelist-search > div > [type="submit"]').click()
+    cy.get(':nth-child(1) > .field-testcase').should(
+      'have.text',
+      `${project_name}: ${testcase_name}`,
```
Comment (gatukgl) [inline tests/cypress/integration/testinstances.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 7]
```diff
@@ -15,116 +15,121 @@ context('Test Runs', () => {
...
+    cy.get(
+      ':nth-child(6) > .table-overview > tbody > tr > :nth-child(3) > .addlink',
+    ).click()
+    cy.get('#id_name').type(`Test new test run ${random_string}`)
     cy.get('#select2-id_project-container').click()
     cy.contains('li', project_name).click()
     cy.get('#select2-id_testinstance_set-0-testcase-container').click()
-    cy.contains('li', testcase_name).click()
-    cy.get('#testinstance_set-0 > .field-assignees > .related-widget-wrapper > .select2 > .selection > .select2-selection > .select2-selection__rendered').click()
+    cy.contains('li', `${project_name}: ${testcase_name}`).click()
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 8]
```diff
@@ -15,116 +15,121 @@ context('Test Runs', () => {
...
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 9]
```diff
@@ -15,116 +15,121 @@ context('Test Runs', () => {
...
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

File: tests/cypress/integration/testruns.spec.js [hunk 10]
```diff
@@ -15,116 +15,121 @@ context('Test Runs', () => {
...
```
Comment (gatukgl) [inline tests/cypress/integration/testruns.spec.js]: This is what I really changed.

**Comment by zkan** [comment]
@gatukgl Conflict!

**Pull request merged**